cashctrl_ledger/ledger.py

```diff
@@ -47,11 +82,120 @@ def add_price():
         """
         raise NotImplementedError
 
-    def add_vat_code():
+    def add_vat_code(
+        self, code: str, rate: float, account: str, inclusive: bool = True,
+        text: str = ""
+    ):
         """
-        Not implemented yet
+        Adds a new VAT code to the CashCtrl account.
+
+        Parameters:
+            code (str): The VAT code to be added.
+            rate (float): The percentage rate of the VAT, must be between 0 and 1.
+            account (str): The account identifier to which the VAT is applied.
+            inclusive (bool): Determines whether the VAT is calculated as 'NET' 
+                            (True, default) or 'GROSS' (False).
+            text (str): Additional text or description associated with the VAT code.
+
+        Raises:
+            ValueError: If any of the inputs are invalid (wrong type or out of allowed range).
+            Exception: If there is an issue with the server connection or the API request.
```

Comment:
I've seen that you've removed the line 'Exception:  ....' in the docstring for `update_vat_code`. That makes sense to me. I suggest to remove it here as well.